crawl-ref/source/webserver/server.py

```diff
@@ -266,12 +270,27 @@ def ensure_tornado_current():
             "You are running a deprecated version of tornado; please update"
             " to at least version 4.")
 
+def usr1_handler(signum, frame):
+    assert signum == signal.SIGUSR1
+    # Support configs without load_games defined
+    if not hasattr(config, 'load_games'):
+        return
+    logging.info("Received USR1, reloading config.")
+    try:
+        config.load_games()
```

Comment:
would be better to trigger `load_games` via `IOLoop.add_callback_from_signal`, rather than calling it directly.

(I'm not 100% sure that this is in the patched tornado 2.4, I seem to remember it needing a compat hack for that case.)

Reply:
Copied the logic from the existing signal handler